README.md

```diff
@@ -44,45 +34,50 @@ If you feel that you may need a lot of schema ids, you can use a single schema
-x5f2        x5f2_status.fbs                   Status update and heartbeat message for any software
-rf5k        rf5k_forwarder_config.fbs         Configuration update for Forwarder
-```
+
+| ID   | File name                        | Description                                                          |
+|------|----------------------------------|----------------------------------------------------------------------|
+| f140 | `f140_general.fbs              ` | [OBSOLETE] Can encode an arbitrary EPICS PV
+| f141 | `f141_ntarraydouble.fbs        ` | [OBSOLETE] A simple array of double, testing file writing
+| f142 | `f142_logdata.fbs              ` | For log data, for example forwarded EPICS PV update
+| f143 | `f143_structure.fbs            ` | [OBSOLETE] Arbitrary nested data
+| ev42 | `ev42_events.fbs               ` | Multi-institution neutron event data for a single pulse
+| ev43 | `ev43_events.fbs               ` | Multi-institution neutron event data from multiple pulses
+| is84 | `is84_isis_events.fbs          ` | ISIS specific addition to event messages
+| ba57 | `ba57_run_info.fbs             ` | [OBSOLETE] Run start/stop information for Mantid [superceded by pl72]
+| df12 | `df12_det_spec_map.fbs         ` | Detector-spectrum map for Mantid
+| senv | `senv_data.fbs                 ` | Used for storing for waveforms from DG ADC readout system.
+| NDAr | `NDAr_NDArray_schema.fbs       ` | (DEPRECATED) Holds binary blob of data with n dimensions.
+| ADAr | `ADAr_area_detector_array.fbs  ` | Holds EPICS area detector array data (in a flatbuffer format).
+| mo01 | `mo01_nmx.fbs                  ` | Daquiri monitor data: pre-binned histograms, raw hits and NMX tracks.
+| ns10 | `ns10_cache_entry.fbs          ` | NICOS cache entry
+| ns11 | `ns11_typed_cache_entry.fbs    ` | NICOS cache entry with typed data
+| hs00 | `hs00_event_histogram.fbs      ` | Event histogram stored in n dim array
+| dtdb | `dtdb_adc_pulse_debug.fbs      ` | Debug fields that can be added to the ev42 schema
+| ep00 | `ep00_epics_connection_info.fbs` | (DEPRECATED) Status of the EPICS connection
+| pvCn | `pvCn_epics_pv_connection.fbs  ` | Status or event of EPICS connection. Replaces _ep00__.
+| json | `json_json.fbs                 ` | Carries a JSON payload
+| tdct | `tdct_timestamps.fbs           ` | Timestamps from a device (e.g. a chopper)
+| pl72 | `pl72_run_start.fbs            ` | File writing, run start message for file writer and Mantid
+| 6s4t | `6s4t_run_stop.fbs             ` | File writing, run stop message for file writer and Mantid
+| answ | `answ_action_response.fbs      ` | Holds the result of a command to the filewriter.
+| wrdn | `wrdn_finished_writing.fbs     ` | Message from the filewriter when it is done writing a file.
+| x5f2 | `x5f2_status.fbs               ` | Status update and heartbeat message for any software
+| rf5k | `rf5k_forwarder_config.fbs     ` | Configuration update for Forwarder
```

Comment:
I guess this table renders prettier? Nice. But it would need to be updated with what has been merged in the meantime.